_includes/devdoc/guide_operating_modes.md

```diff
@@ -20,7 +20,7 @@ Currently there are two primary methods of validating the block chain as a clien
 
 The first and most secure model is the one followed by Bitcoin Core, also known as a “thick” or “full chain” client. This security model assures the validity of the block chain by downloading and validating blocks from the genesis block all the way to the most recently discovered block. This is known as using the *height* of a particular block to verify the client’s view of the network. 
 
-For a client to be fooled, an adversary would need to give a complete alternative block chain history that is of greater difficulty than the current “true” chain, which is impossible due to the fact that the longest chain is by definition the true chain. After the suggested six confirmations, the ability to fool the client become intractable, as only a single honest network node is needed to have the complete state of the block chain. 
+For a client to be fooled, an adversary would need to give a complete alternative block chain history that is of greater difficulty than the current “true” chain, which is computationally expensive (if not impossible) due to the fact that the chain with the most cumulative proof of work is by definition the "true" chain. Due to the computational difficulty required to generate a new block at the tip of the chain, the ability to fool a full node become intractable after 6 confirmations. This form of verification is highly resistent to sybil attacks - only a single honest network peer is required in order to receive and verify the complete state of the "true" block chain. 
```

Comment:
@jlopp 

- "fool a full node **become** intractable" should be **becomes**

- For `sybil attacks - only a single` could you make that `sybil attacks---only a single` (the triple dash will be transformed by our Markdown interpreter into a proper emdash)

Otherwise, this LGTM.  CC: @instagibbs (who I believe authored this paragraph)